src/aind_ephys_ibl_gui_conversion/ephys.py

```diff
@@ -488,233 +450,247 @@ def get_ecephys_stream_names(base_folder: Path) -> tuple[list[str], Path, int]:
+                logging.info("Applying high pass filter to AP stream")
+                recording_group_ap_highpass = spre.highpass_filter(
+                    recording_group
+                )
+
+                recording_lfp, is_1_0_probe = get_neuropixel_lfp_stream(
+                    recording_group,
+                    stream_name,
+                    ecephys_compressed_folder,
+                    block_index,
+                )
+                recording_lfp_processed = process_lfp_stream(
+                    recording_lfp,
+                    is_1_0_probe,
+                    freq_min,
+                    freq_max,
+                    decimation_factor,
+                )
+
+                # assume this is a surface finding recording
+                if recording_time < min_duration_secs:
+                    surface_recordings_ap[stream_name].append(
+                        recording_group_ap_highpass
+                    )
+                    surface_recordings_lfp[stream_name].append(
+                        recording_lfp_processed
+                    )
                 else:
-                    if key not in main_recordings:
-                        main_recordings[key] = [recording_group]
-                    else:
-                        main_recordings[key].append(recording_group)
-
-                if has_lfp:
-                    key = key.replace("AP", "LFP")
-
-                    _reset_recordings(recording_groups_lfp[group], key)
-                    if (
-                        recording_groups_lfp[group].get_total_duration()
-                        < min_duration_secs
-                    ):
-                        if key not in recording_mappings:
-                            recording_mappings[key] = [
-                                recording_groups_lfp[group]
-                            ]
-                        else:
-                            recording_mappings[key].append(
-                                recording_groups_lfp[group]
-                            )
-                    else:
-                        if key not in main_recordings:
-                            main_recordings[key] = [
-                                recording_groups_lfp[group]
-                            ]
-                        else:
-                            main_recordings[key].append(
-                                recording_groups_lfp[group]
-                            )
+                    main_recordings_ap[stream_name].append(
+                        recording_group_ap_highpass
+                    )
+                    main_recordings_lfp[stream_name].append(
+                        recording_lfp_processed
+                    )
 
-    return main_recordings, recording_mappings
+    return (
+        main_recordings_ap,
+        surface_recordings_ap,
+        main_recordings_lfp,
+        surface_recordings_lfp,
+    )
 
 
-def _save_rms_and_lfp_spectrum(
+def save_rms_and_lfp_spectrum(
```

Comment:
can we add some `logging` to report elapsed time for `compute_rms` and the compute psd part? So we can quantify performance improvements easily!